src/main/java/frc/trigon/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,89 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.sysid.SysIdRoutineLog;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.hardware.phoenix6.talonfx.TalonFXMotor;
+import frc.trigon.robot.hardware.phoenix6.talonfx.TalonFXSignal;
+import frc.trigon.robot.subsystems.MotorSubsystem;
+import frc.trigon.robot.utilities.Conversions;
+import frc.trigon.robot.utilities.ShootingCalculations;
+
+public class Shooter extends MotorSubsystem {
+    private final ShootingCalculations shootingCalculations = ShootingCalculations.getInstance();
+    private final TalonFXMotor
+            rightMotor = ShooterConstants.RIGHT_MOTOR,
+            leftMotor = ShooterConstants.LEFT_MOTOR;
+
+    private final VelocityVoltage velocityRequest = new VelocityVoltage(0).withEnableFOC(ShooterConstants.FOC_ENABLED);
+    private double
+            rightMotorTargetVelocityRotationsPerSecond = 0,
+            leftMotorTargetVelocityRotationsPerSecond = 0;
+
+    public Shooter() {
+        setName("Shooter");
+    }
+
+    public void updateLog(SysIdRoutineLog log) {
+        log.motor("RightShooter")
+                .linearPosition(Units.Meters.of(toMeters(rightMotor.getSignal(TalonFXSignal.POSITION))))
+                .linearVelocity(Units.MetersPerSecond.of(rightMotor.getSignal(TalonFXSignal.VELOCITY)))
```

Comment:
the status signal returns velocity in rotations per second, but ur units are meters per second